packages/oca/src/types/branding/BrandingOverlay.ts

```diff
@@ -3,16 +3,16 @@ import { generateColor } from '../../utils/color'
 import BaseOverlay from '../base/BaseOverlay'
 
 export default class BrandingOverlay extends BaseOverlay {
-  #background_image: string
-  #background_image_slice: string
+  #background_image?: string
+  #background_image_slice?: string
   #primary_background_color: string
```

Comment:
This should be optional too?

Reply:
I think it should, thinking about a bundle which only defines the bg image or the slice one (or none and just sets the color for example)